### DISCUSSION TITLE
The safe exam browser cannot be initialized properly, it always gets stuck on this screen

### DISCUSSION THREAD

**Discussion by andersonpson** [SafeExamBrowser/seb-win-refactoring#481] [Q&A]
![IMG_2120](https://user-images.githubusercontent.com/115115559/194162871-d60c8b10-be4e-4a89-96b4-3b55ab8c5632.jpg)
[2022-10-05_17h08m01s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/9718954/2022-10-05_17h08m01s_Runtime.log)

**Comment by andersonpson** [comment]
[2022-10-05_17h08m01s_Browser.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/9718995/2022-10-05_17h08m01s_Browser.log)
[2022-10-05_17h08m01s_Client.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/9718996/2022-10-05_17h08m01s_Client.log)

**Reply by dbuechel** [reply]
I unfortunately do not speak Chinese, would you mind translating on which initialization step it gets stuck? The runtime log does show that there appear to be at least two issues:

```
2022-10-05 17:43:51.307 [19] - WARNING: [DesktopMonitor] Failed to get currently active desktop!
2022-10-05 17:43:59.214 [23] - DEBUG: [Process 'SafeExamBrowser.Client.exe' (11524)] Process has terminated.
2022-10-05 17:43:59.214 [08] - ERROR: Client instance terminated unexpectedly during initialization!
2022-10-05 17:43:59.215 [08] - ERROR: Failed to start new client instance! Aborting procedure...
```

Unfortunately, the exact reason of failure is not visible in the application logs itself. Could you please post the respective error logs from the Windows event viewer (to be found around the timestamp posted above under "Windows Logs" > "Application").

**Reply by andersonpson** [reply]
the setp i think is “Initialize the browser”

I can explain what happened, since the browser had been initialized for close to 30 minutes, I had to call out the task manager to forcibly kill it

**Reply by andersonpson** [reply]
It's weird that I've had one or two successful runs out of a large number of failed runs, but after a few hours I'm stuck on the initializer screen again

**Reply by dbuechel** [reply]
That is weird indeed. Did you install SEB using the setup (.msi) or setup bundle (.exe)? It might be that you don't have all the required dependencies installed as listed in the README here on GitHub.

Could you find any errors or warnings in the event viewer of Windows itself?

**Comment by andersonpson** [comment]
Hi, I used verificator to check and it shows like this:
![image](https://user-images.githubusercontent.com/115115559/194350152-5612613a-5362-425f-9897-0b385c13e32f.png)

I used the setup bundle when installing

Sorry I can't find anything because for some reason my computer is set to "do not store records"
![image](https://user-images.githubusercontent.com/115115559/194351112-83980b9c-992f-4dbf-b006-218cc3b41288.png)
I only found this, don't know if it helps

**Reply by dbuechel** [reply]
The verificator only verifies whether the SEB installation has been corrupted / manipulated, not whether all required dependencies are correctly installed.

> I can't find anything because for some reason my computer is set to "do not store records"

This might be the root of the cause. What do you exactly mean with "my computer is set to 'do not store records'"? Where is that setting specified? It appears to be the case then that the browser engine fails to initialize, if I could hazard a guess most likely due to some configuration setting(s) on your system.
